tests/conftest.py

```diff
@@ -19,11 +19,24 @@ def remove_headers(response: dict):
 
 
 @pytest.fixture(scope="session")
-def token() -> Optional[str]:
-    return os.getenv("NOTION_TOKEN")
+def token() -> str:
+    notion_token = os.environ.get("NOTION_TOKEN")
+    if not notion_token:
+        raise EnvironmentError("Env variable NOTION_TOKEN needed for testing")
+    return notion_token
 
 
-@pytest.fixture
+@pytest.fixture(scope="session")
+def testing_page() -> str:
```

Comment:
Let's make the fixture name and environment variable name consistent so that it's easier to read and understand. :)

Reply:
Also, is it really a URL or just the page id?

Reply:
A page id, I'll rename it as NOTION_TESTING_PAGE_ID for clarity

Reply:
LGTM (although I think you can drop the ING :P)